apps/cookbook/src/app/showcase/data-table-showcase/data-table-showcase.component.html

```diff
@@ -28,10 +26,30 @@ <h2>Data table with minimal data</h2>
   </cookbook-example-viewer>
 
   <h2>Data table wrapped with Kirby Card</h2>
-  <p>Data table wrapped with card and with selectable rows.</p>
+  <p>Data table wrapped with card, with selectable rows.</p>
   <cookbook-example-viewer [html]="dataTableCardExample.template">
     <cookbook-data-table-card-example #dataTableCardExample></cookbook-data-table-card-example>
   </cookbook-example-viewer>
+
+  <h2>Sortable table</h2>
```

Comment:
Add link to sort examples on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#sorting_array_of_objects